tests/integration/users/test_views.py

```diff
@@ -0,0 +1,86 @@
+import json
+
+from openslides.utils.test import TestCase
+
+
+class TestWhoAmIView(TestCase):
+    url = '/users/whoami/'
+
+    def test_get_anonymous(self):
+        response = self.client.get(self.url)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.content, b'{"user_id": null}')
```

Comment:
What do you mean with response.data? When I try it I get an Error:
```
AttributeError: 'HttpResponse' object has no attribute 'data
```
I do not like to decode the return value, because I want to test the exact response of the view and not a result that was altert.